llarp/handlers/tun.cpp

```diff
@@ -940,6 +807,25 @@ namespace llarp
           dst = pkt.dstv6();
           src = pkt.srcv6();
         }
+        /*
```

Comment:
if this commented block of code makes sense to include, at least put a TODO or FIXME or something as well.